PokeDemo/Models/Sprite.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Sprite.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 06/07/22.
+//
+
+import Foundation
+
+struct Sprite: Decodable {
```

Comment:
What is Sprite?

Reply:
All pokemon detail is returned in response with this key, so kept it's name same

Reply:
Change the name. It doesnot mean to our project models. Learn about domain models. Check article in JSON decoder to keep separate names that doesn't match JSON response key

Reply:
Covered in f05532612b67d77b533e2e3503714f79ff96df2d